README.md

```diff
@@ -22,9 +22,10 @@
 
```

Comment:
- [ ] Please you should record a video presenting the features of the project:-
1. The maximum length of the video is 5 minutes.
1. The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
1. The video contains a demo of the project, highlighting all the project features.
1. The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

>

Reply:
Hi Meqdam. Thank you for the review. I already recorded the video and included a link to it in the README, called 'Project description.  Did you see it, and if so, do I need to redo it?

Reply:
@Thoth1111 Sorry for the confusion, but you should make the name more clear so the person who reviews your project can see it easily

Reply:
Okay, thank you @MeqdamAlqudah. No worries,  I've renamed in 'video presentation' in my latest commit.